student-test/assignment1/Test_validate_username.c

```diff
@@ -18,5 +18,10 @@ void test_validate_my_username()
      * TODO: Replace the line below with your code here as described above to verify your /conf/username.txt 
      * config file and my_username() functions are setup properly
      */
-    TEST_ASSERT_TRUE_MESSAGE(false,"AESD students, please fix me!");
+    //TEST_ASSERT_TRUE_MESSAGE(false,"AESD students, please fix me!");
+    char* temp_buff = malloc_username_from_conf_file();
+    TEST_ASSERT_EQUAL_STRING_MESSAGE(my_username(), temp_buff, "Username Validated");
```

Comment:
@ruchitnaik the purpose of message here is to report an error of username mismatch.

Reply:
Can you please elaborate? The purpose of this execution was to free the malloc space after being used in assert.

Reply:
purpose of TEST_ASSERT_EQUAL_STRING_MESSAGE was to report an error on username mismatch. 3rd argument is a message which will be printed on mismatch of username comparison.

Reply:
Noted. I will take care of it in the next assignment. Thanks for pointing it out.